Sources/RingPublishingTracking/Private/Tracking/KeepAlive/KeepAliveManager.swift

```diff
@@ -39,6 +39,7 @@ final class KeepAliveManager {
     private var pauseTimeStart = [Date]()
     private var pauseTimeEnd = [Date]()
 
+    private let timerQueue = DispatchQueue(label: "timerQueue", attributes: .concurrent)
```

Comment:
To jest fix na testy które na fastlanie nie przechodziły. Timery były uruchamiane z qos: .background. Teraz zrobiłem dedykowaną kolejkę .concurrent